messages/ownship/pilot_card.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
I think tat the contents of this file can be split into different observation messages. For example, an observation message called `voyage_information` and another called `vessel_description`. Either way, they should be in `observations`, although that they are not strictly speaking from a sensor.

Reply:
I think this ties in well with my previous comment on having a discussion regarding an entity-wide configuration description. Lets set up a discussion regarding this.